tests/test_document.py

```diff
@@ -127,70 +138,45 @@ def test_get_by_uri(self):
         assert doc
         assert_equal(doc['title'], "document1") 
 
-    def test_get_all_by_uri(self):
+    @staticmethod
+    def test_get_all_by_uri():
```

Comment:
Actually, you have a test like this later. I would just remove this test. It's not public.